### PR TITLE
CPP-2824 Add cpp helpers

### PR DESCRIPTION
**Pull request by nitinsinha1**
What does this PR do?
----------------------
Add required CPP helpers which are commonly used in other CPP-services.

Notes
----------------------
 - Could not use latest jest version as some unit test require code changes in order to use version ^23
 - Very specific the service modules have not been included in cpp-helpers (reviewEventStream, middyMiddleware, fieldMapper, reviewStatusHelper, queryParser, massageData)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 const invokeLambda = require('./lib/invokeLambdaHandler')
+const general = require('./lib/general')
+const eventStreamHelper = require('./lib/eventStreamHelper')
+const sqsHelper = require('./lib/sqsHelper')
+const jwtHelper = require('./lib/jwtHelper')
 
 module.exports = {
     dynamoDbHelper,
-    invokeLambda
+    invokeLambda,
```
Comment (essame) [inline index.js]: What do you think about adding `Helper` suffix to all modules?

Comment (essame) [reply index.js]: Other than that, looks good!

Comment (nitinsinha1) [reply index.js]: Thanks Essam, I will update the name and keep it consistent.

Comment (essame) [reply index.js]: @nitinsinha1 forgot to push? : D

Comment (nitinsinha1) [reply index.js]: sorry forgot to push :D have done it now.

**Pull request merged**